gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorPage.kt

```diff
@@ -49,21 +65,42 @@ internal fun GravatarQuickEditorPage(
                 }
             }
         }
-        composable(QuickEditorPage.OAUTH.name, enterTransition = { EnterTransition.None }) {
+        composable(route = QuickEditorPage.OAUTH.name, enterTransition = { EnterTransition.None }) {
             OAuthPage(
                 oAuthParams = oAuthParams,
                 email = gravatarQuickEditorParams.email,
                 onAuthError = { onDismiss(GravatarQuickEditorDismissReason.OauthFailed) },
                 onAuthSuccess = { navController.navigate(QuickEditorPage.EDITOR.name) },
             )
         }
-        composable(QuickEditorPage.EDITOR.name) {
-            AvatarPicker(
-                gravatarQuickEditorParams = gravatarQuickEditorParams,
-                handleExpiredSession = true,
-                onAvatarSelected = onAvatarSelected,
-                onSessionExpired = { navController.navigate(QuickEditorPage.OAUTH.name) },
-            )
+        navigation(
+            route = QuickEditorPage.EDITOR.name,
+            startDestination = EditorNavDestinations.AVATAR_SELECTION.name,
+        ) {
+            composable(route = EditorNavDestinations.AVATAR_SELECTION.name) {
```

Comment:
💡 In a separate PR, we could work a bit on the enter/exit animations.

Reply:
I already have a [ticket](https://github.com/Automattic/Gravatar-SDK-Android/issues/498) for that 😃